tests/FVTTests/fileParameterView/testFileParameterView.ts

```diff
@@ -102,13 +106,7 @@ describe('JES explorer spool file in url query (explorer-jes/#/viewer)', functio
 
         const [jobId, fileName] = cardHeaderText.split('-');
         expect(jobId).to.be.equal(testFilters.jobId);
-        const testFileName = 'JESJCL';
         expect(fileName).to.be.equal(testFileName);
```

Comment:
As testFileName constant is only used within this single test the declaration and scope of the constant should be within this test

Reply:
Sure. I will move this constant declaration to this test.